### DISCUSSION TITLE
[Unknown Mod][Game] Recruit window, recruitment cost of all available creatures is displayed in this window

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#5652] [Ideas]
The recruitment cost of all available creatures is displayed in the Recruit window.
In the example, to know at a glance how much gold to trade in the Marketplace to be able to recruit all the Grand Elves, in this case 5400 gold.

For example, on the same line as the text "Available: 18" and centered horizontally with respect to the "Cost per troop" frame:

"Total cost: 5400"

In fheroes2:

![Total creatures cost 01](https://user-images.githubusercontent.com/43583994/179035673-1eb87738-0679-442e-854c-ff141c48ae46.png)

This is not an option.
This can be implemented directly in fheroes2.

**Comment by Branikolog** [comment]
We have a separate block with "Cost per troop" phrase and even gold pile inside this window to show the price for a single troop. Placing the same type info (Total cost) in a simple poor-looking text line in a small area between blocks looks not optimal and not pleasant to my eye...
I think, it will look better, if we manage to place total cost inside this box with gold pile...
Like: "300 (5400 total)"
To better correspond the inner provided information, we can change the header of the box. Something like "Hiring cost" or "Pay for hiring". I'm not sure, the text is not a part of the box, but it seems not really hard to replace it.

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> We have a separate block with "Cost per troop" phrase and even gold pile inside this window to show the price for a single troop. Placing the same type info (Total cost) in a simple poor-looking text line in a small area between blocks looks not optimal and not pleasant to my eye... I think, it will look better, if we manage to place total cost inside this box with gold pile... Like: "300 (5400 total)" To better correspond the inner provided information, we can change the header of the box. Something like "Hiring cost" or "Pay for hiring". I'm not sure, the text is not a part of the box, but it seems not really hard to replace it.

The case of creatures that require resources should also be taken into account.

![Recruit window total cost 01](https://user-images.githubusercontent.com/43583994/179539704-b8c5303e-b885-4f79-94b2-5fe91f552c8f.png)

Maybe just "3000 (9000)... ...1 (3)" without the word "total" then.
I find that the word total will "confuse" the reading.
Or maybe add an extra line inside the yellow frame (vertically enlarged) "Total: 9000... ...3".
